routers/web/web.go

```diff
@@ -612,6 +612,7 @@ func RegisterRoutes(m *web.Route) {
 				m.Combo("/*").Get(repo.SettingsProtectedBranch).
 					Post(bindIgnErr(forms.ProtectBranchForm{}), context.RepoMustNotBeArchived(), repo.SettingsProtectedBranchPost)
 			}, repo.MustBeNotEmpty)
+			m.Post("/rename_branch", bindIgnErr(forms.RenameBranchForm{}), context.RepoMustNotBeArchived(), repo.RenameBranchPost)
```

Comment:
As far as I have seen, the underscore (`_`) syntax is uncommon for routes. What about `/branches/rename` instead? That would then be
```suggestion
				m.Combo("/*").Get(repo.SettingsProtectedBranch).
					Post(bindIgnErr(forms.ProtectBranchForm{}), context.RepoMustNotBeArchived(), repo.SettingsProtectedBranchPost)
				m.Post("/rename", bindIgnErr(forms.RenameBranchForm{}), context.RepoMustNotBeArchived(), repo.RenameBranchPost)
			}, repo.MustBeNotEmpty)
```
Combined with a change inside the UI dialog.

Reply:
well I dont mind since it's only a web route - if you like to group settings bit more refactor pulls are welcome :)